website/build.gradle

```diff
@@ -65,8 +65,12 @@ task createDockerContainer(type: Exec) {
   ext.containerId = {
     return standardOutput.toString().trim()
   }
+  def extraOptions = ''
+  if (project.hasProperty('publishJekyllPort')) {
+    extraOptions = '-p 127.0.0.1:4000:4000'
```

Comment:
Can you add a quick comment about what this does? I assume website contributors are not all Docker fluent (I'm not).

[Docker docs](https://docs.docker.com/v17.09/engine/reference/commandline/create/) describe it as "Publish a container’s port(s) to the host", although I don't find that totally intuitive.

Would it be correct to say something like: "Specify `-PpublishJekyllPort` on the Gradle commandline to serve the website locally at http://localhost:4000" ?

Is my understanding correct that if this option isn't specified, the served website is inaccessible? That seems not very usable.

Reply:
I'm not that Gradle fluent. :)
What I wanted was to set this option for createDockerContainer, but only if gradle is asked to run task serveWebsite. So the option wouldn't be set on pre/post-commits, where it might cause them to fail (if port 4000 is already taken).

Reply:
Sounds good; we can revisit this later if it's a burden.